apps/src/weblab/WebLab.js

```diff
@@ -161,7 +160,10 @@ WebLab.prototype.init = function (config) {
   }
 
   function onAddFileImage() {
-    dashboard.assets.showAssetManager(null, 'image', this.loadCurrentAssets.bind(this), !this.studioApp_.reduxStore.getState().pageConstants.is13Plus);
+    dashboard.assets.showAssetManager(null, 'image', this.loadFileEntries.bind(this), {
+      showUnderageWarning: !this.studioApp_.reduxStore.getState().pageConstants.is13Plus,
+      useFilesApi: true
```

Comment:
`config.useFilesApi` is needed to impact version history (which is invoked from within `studioApp`). We can switch off it here as well, but since we're invoking `showAssetManager` directly, I'm not sure that it adds much value...